Author: Artjom Slepnjov "Shellgen"
Description: Quit a verbose output
Ver: ld-preload-socket v2020-09-29 or newer version.
Date: 2026-10-05 11:00 UTC
==============================================================
--- a/ld-preload-socket.c
+++ b/ld-preload-socket.c
@@ -155,7 +155,6 @@
         return;
     }
 
-    fprintf(stderr, "%s: Mapping AF_UNIX %s(%s) to %s(%s)\n", LOG_PREFIX, func_name, addr->sun_path, func_name, mapped_path);
     strncpy(addr->sun_path, mapped_path, sizeof(addr->sun_path));
     addr->sun_path[sizeof(addr->sun_path) - 1] = '\0';
 }
@@ -239,7 +238,6 @@
         return;
     }
 
-    fprintf(stderr, "%s: Mapping AF_INET %s(%d) to %s(%d)\n", LOG_PREFIX, func_name, source_port, func_name, target_port);
     addr->sin_port = htons(target_port);
 }
 
@@ -253,7 +251,6 @@
         return;
     }
 
-    fprintf(stderr, "%s: Mapping AF_INET6 %s(%d) to %s(%d)\n", LOG_PREFIX, func_name, source_port, func_name, target_port);
     addr->sin6_port = htons(target_port);
 }
 
@@ -306,7 +303,6 @@
     const char *mapped_pathname = map_unix_socket_path(pathname);
     if (mapped_pathname != NULL)
     {
-        fprintf(stderr, "%s: Mapping AF_UNIX unlink(%s) to unlink(%s)\n", LOG_PREFIX, pathname, mapped_pathname);
         path_to_unlink = mapped_pathname;
     }
 
@@ -323,19 +319,10 @@
     {
         parse_unix_socket_paths();
     }
-    fprintf(stderr, "%s: %d mappings defined for UNIX socket paths\n", LOG_PREFIX, unix_socket_paths_len);
     for (size_t i = 0; i < unix_socket_paths_len; i++)
-    {
-        fprintf(stderr, "%s:    '%s' -> '%s'\n", LOG_PREFIX, unix_socket_paths[i].source, unix_socket_paths[i].target);
-    }
 
     if (inet_socket_ports == NULL)
     {
         parse_inet_socket_ports();
     }
-    fprintf(stderr, "%s: %d mappings defined for INET socket ports\n", LOG_PREFIX, inet_socket_ports_len);
-    for (size_t i = 0; i < inet_socket_ports_len; i++)
-    {
-        fprintf(stderr, "%s:    %d -> %d\n", LOG_PREFIX, inet_socket_ports[i].source, inet_socket_ports[i].target);
-    }
 }
